src/components/Tile/Tile.styles.ts

```diff
@@ -15,12 +17,13 @@ const mapAspectRatioToPercent: Record<TileAspectRatio, string> = {
 };
 
 export const getClassNames = memoizeFunction((classNameProps: TileClassNameProps) => {
-  const { aspectRatio, theme } = classNameProps;
+  const { aspectRatio, borderColor, theme } = classNameProps;
+
+  const border = borderColor ? `solid 1px ${theme.palette[borderColor]}` : undefined;
+  const shift = borderColor ? 1 : 0;
```

Comment:
I think we should extract a borderWidth variable to hold 1, and use that in the `border` template string, so the two are linked. (Since it may not be immediately apparent that the shift value corresponds to the border thickness)

Reply:
Fixed!